Description: Remove upstream setup dependencies.
 Tahoe-lafs setup uses a custom setuptools which rely on darcsver to do some
 smart guessings. This isn't relevant in Debian and needs to be ignored.
Author: bertagaz <bertagaz@ptitcanardnoir.org>
Index: tahoe/setup.py
===================================================================
--- tahoe.orig/setup.py	2012-07-04 21:41:43.295659068 +0200
+++ tahoe/setup.py	2012-07-04 21:42:07.535159561 +0200
@@ -68,13 +68,13 @@
     del sys.argv[1]
     install_requires += ["fakedependency >= 1.0.0"]
 
-__requires__ = install_requires[:]
+#__requires__ = install_requires[:]
 
-egg = os.path.realpath(glob.glob('setuptools-*.egg')[0])
-sys.path.insert(0, egg)
-egg = os.path.realpath(glob.glob('darcsver-*.egg')[0])
-sys.path.insert(0, egg)
-import setuptools; setuptools.bootstrap_install_from = egg
+#egg = os.path.realpath(glob.glob('setuptools-*.egg')[0])
+#sys.path.insert(0, egg)
+#egg = os.path.realpath(glob.glob('darcsver-*.egg')[0])
+#sys.path.insert(0, egg)
+#import setuptools; setuptools.bootstrap_install_from = egg
 
 from setuptools import setup
 from setuptools.command import sdist
@@ -476,7 +476,7 @@
       package_data={"allmydata.web": ["*.xhtml"],
                     "allmydata.web.static": ["*.js", "*.png", "*.css"],
                     },
-      setup_requires=setup_requires,
+      #setup_requires=setup_requires,
       entry_points = { 'console_scripts': [ 'tahoe = allmydata.scripts.runner:run' ] },
       zip_safe=False, # We prefer unzipped for easier access.
       versionfiles=['src/allmydata/_version.py',],
Index: tahoe/setup.cfg
===================================================================
--- tahoe.orig/setup.cfg	2012-07-04 21:41:43.295659068 +0200
+++ tahoe/setup.cfg	2012-07-04 21:58:29.839700075 +0200
@@ -9,12 +9,3 @@
 tag_date = 0
 tag_svn_revision = 0
 
-[aliases]
-sdist_dsc = update_version sdist_dsc
-sdist = update_version sdist
-trial = update_version trial
-build = update_version develop --prefix=support make_executable build
-install = update_version install
-test = update_version develop --prefix=support make_executable build trial
-bdist_egg = update_version bdist_egg
-
